templates/Pages/MasterDetail.CaliburnMicro/Views/MasterDetailViewPage.xaml

```diff
@@ -38,10 +38,11 @@
             Grid.Row="1"
             x:Name="MasterDetailsViewControl"
             ItemsSource="{x:Bind ViewModel.Items}"
-            SelectedItem="{x:Bind ViewModel.ActiveItem}"
+            SelectedItem="{x:Bind ViewModel.ActiveItem, Mode=OneWay}"
```

Comment:
Thanks, for catching this, one time definitely wasn't right.  Although in my opinion this should be a two way binding, typically other code in the view model should be based off `ActiveItem` and that it should remain in sync with what the user has selected.